src/tls/test-server.c

```diff
@@ -807,13 +796,9 @@ main (int argc, char *argv[])
               setup, test_tls_client_cert_parallel, teardown);
   g_test_add ("/server/tls/client-cert-parallel/alternate", TestCase, &fixture_alternate_client_cert,
               setup, test_tls_client_cert_parallel, teardown);
-  g_test_add ("/server/tls/combined-server-cert-key", TestCase, &fixture_combined_crt_key,
-              setup, test_tls_no_client_cert, teardown);
-  g_test_add ("/server/tls/cert-chain", TestCase, &fixture_cert_chain,
-              setup, test_tls_cert_chain, teardown);
   g_test_add ("/server/tls/no-server-cert", TestCase, NULL,
               setup, test_tls_no_server_cert, teardown);
-  g_test_add ("/server/tls/redirect", TestCase, &fixture_combined_crt_key,
+  g_test_add ("/server/tls/redirect", TestCase, &fixture_separate_crt_key,
```

Comment:
Let's also drop the "separate" name part then, as there is no "combined" any more.

Reply:
I agree.  Will come back to that.